datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/config/BackfillCorpUserInfoConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.linkedin.datahub.upgrade.config;
```

Comment:
No need for a backfill. Use the same logic as soft delete, where if it doesn't exist then it is not a system user by making the boolean optional.

Reply:
Added a backfill job because the field I added to PDL had a searchable annotation, wanted to make sure all corpUser documents in elastic were updated so that we could filter users by whether they are system users or not.

Reply:
By default all users should be non-system. This status should rely on non-existence as false. The PDL annotation should be an optional boolean to support this.